processing/src/main/java/io/druid/segment/CompressedPools.java

```diff
@@ -35,7 +36,7 @@
 {
   private static final Logger log = new Logger(CompressedPools.class);
 
-  public static final int BUFFER_SIZE = 0x10000;
+  public static final int BUFFER_SIZE = LZFChunk.MAX_CHUNK_LEN;
```

Comment:
I know @xvrl choose this buffer size as a power of 2 for optimization purposes. I'd be surprised if this were causing unit tests to break...